kubric/randomness.py

```diff
@@ -67,17 +67,8 @@ def position_sampler(region):
   region = np.array(region, dtype=np.float)
```

Comment:
Missing docstring (and type annotations)?
(or at least an assert on the shape of region)
I would have no idea what to pass to this method.

Reply:
Agreed. But is documenting every function really a priority for us ATM? Because there are tons of functions without docstrings, and I can easily spend days tidying that up.